kfs-core/src/main/resources/edu/arizona/kfs/tax/spring-tax.xml

```diff
@@ -33,4 +33,9 @@
 			</list>
 		</property>
 	</bean>
+
+	<bean id="taxParameterHelperDao" parent="platformAwareDao" class="edu.arizona.kfs.tax.dataaccess.impl.TaxParameterHelperDaoImpl" />
+
+	<bean id="taxParameterHelperService" class="edu.arizona.kfs.tax.service.impl.TaxParameterHelperServiceImpl" /> 
```

Comment:
Last I heard, use of the "lazy service loaders" was preferred by Kuali because it reduces active memory use and reduces server startup time. I can certainly refactor this to inject these services if you want.